core/includes/pager.inc

```diff
@@ -230,9 +230,11 @@ function theme_pager($variables) {
           );
         }
         if ($i == $pager_current) {
+          $pager_current_parameters = $parameters;
+          $pager_current_parameters['aria-current'] = "page";
           $items[] = array(
             'class' => array('pager-current'),
-            'data' => $i,
+            'data' => theme('pager_link', array('text' => $i, 'page_name' => pager_load_array($i - 1, $element, $pager_page_array), 'element' => $element, 'parameters' => $pager_current_parameters)),
```

Comment:
I don't think this should be a link, since it's the current page marker (and doesn't link to any other page). Why not just use `'<span aria-current="page">' . $i . '</span>'`

Reply:
Sounds good! Addressed in the new commit.